src/System.Reflection.TypeExtensions/ref/Configurations.props

```diff
@@ -2,9 +2,9 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
+      netstandard;
       netfx;
-      uap;
+      net461;
```

Comment:
BuildConfigurations should be a superset of PackageConfigurations.  PackageConfigurations is only needed when we don't want to include one of the BuildConfigurations in PackageConfigurations.  In the case of `netfx` that is automatically excluded.